doc/advanced/content/pcl_style_guide.rst

```diff
@@ -50,23 +50,21 @@ same directory, in any other case the include statement is made with
 1.4. Defines & Macros
 ^^^^^^^^^^^^^^^^^^^^^
 
-Macros should all be **ALL_CAPITALS_AND_UNDERSCORED**. Defines for header type
-files also need a trailing underscore. Their naming should be mapped from their
-include name: ``pcl/filters/bilateral.h`` becomes ``PCL_FILTERS_BILATERAL_H_``.
-The ``#ifndef`` and ``#define`` lines should be placed just past the BSD license.
-The ``#endif`` goes all the way at the bottom and needs to have the define name in
-its comment, e.g:
+Macros should all be **ALL_CAPITALS_AND_UNDERSCORED**.
+
+Include guards are not implemented with defines, instead ``#pragma once`` should be used.
 
  .. code-block:: cpp
 
   // the license
   
-  #ifndef PCL_MODULE_NAME_IMPL_FILE_NAME_HPP_
-  #define PCL_MODULE_NAME_IMPL_FILE_NAME_HPP_
+  #pragma once
   
-  // the code
+  // includes
   
-  #endif // PCL_MODULE_NAME_IMPL_FILE_NAME_HPP_
+  #define MY_CUSTOM_DEFINE 2.5f
```

Comment:
Nitpicking a little here but can you remove this line. I don't want to convey the idea that it is OK to define constants at global scope in pcl headers.

Reply:
Remove the whole line please. We usually don't use global definitions unless for very specific things like the ones you can find in [pcl_config.h.in](https://github.com/PointCloudLibrary/pcl/blob/master/pcl_config.h.in). 

With that example in mind, the most common case to appear at this level is
```suggestion
  namespace pcl
  {
    class FileName
    {
      [...]
    };
  }
```